examples/nlp/semantic_similarity_with_bert.py

```diff
@@ -0,0 +1,363 @@
+        return len(self.sentence1) // self.batch_size
+
+    def __getitem__(self, idx):
+        # Generates batch of data
+        indexes = self.indexes[idx * self.batch_size : (idx + 1) * self.batch_size]
+        sentence1 = self.sentence1[indexes]
+        sentence2 = self.sentence2[indexes]
+        batch_input_ids = []
+        batch_attention_masks = []
+        batch_token_type_ids = []
+        # encoding both the sentences together. With
+        # BERT tokenizer's encode plus both the sentences are
+        # encoded together and separated by [SEP] token.
+        # here we are encoding batch of sentences together.
+        for s1, s2 in zip(sentence1, sentence2):
+            encoded = self.tokenizer.encode_plus(
+                s1,
+                s2,
+                add_special_tokens=True,
+                max_length=MAXLEN,
+                return_attention_mask=True,
+                return_token_type_ids=True,
+                padding=True,
+                pad_to_max_length=True,
+                pad_to_multiple_of=MAXLEN,
+                return_tensors="tf",
+            )
+            batch_input_ids.extend(encoded["input_ids"])
+            batch_attention_masks.extend(encoded["attention_mask"])
+            batch_token_type_ids.extend(encoded["token_type_ids"])
+
+        # convert batch of encoded features to numpy arary
+        input_ids = np.array(batch_input_ids, dtype="int32")
+        masks = np.array(batch_attention_masks, dtype="int32")
+        token_type_ids = np.array(batch_token_type_ids, dtype="int32")
+
+        # set to true if data generator is used for training/validation
+        if self.train:
+            labels = np.array(self.labels[indexes], dtype="int32")
+            return [input_ids, masks, token_type_ids], labels
+        else:
+            return [input_ids, masks, token_type_ids]
+
+    def on_epoch_end(self):
+        # shuffle indexes after each epoch if self.shuffle is set to True
+        self.indexes = np.arange(len(self.sentence1))
+        if self.shuffle:
+            np.random.shuffle(self.indexes)
+
+
+"""
+## Build The Model
+"""
+
+
+def build_model():
+    """
+    model inputs:
+                  input_ids:       Encoded token Ids from BERT tokenizer.
+                  attention_masks: This argument indicates to the model which
+                                   tokens should be attended to.
+                  token_type_ids:  They are binary masks identifying different
+                                   sequences in the model.
+    """
+    input_ids = tf.keras.layers.Input(shape=(MAXLEN,), dtype=tf.int32, name="input_ids")
+    attention_masks = tf.keras.layers.Input(
+        shape=(MAXLEN,), dtype=tf.int32, name="att_mask"
+    )
+    token_type_ids = tf.keras.layers.Input(
+        shape=(MAXLEN,), dtype=tf.int32, name="tt_ids"
+    )
+    # Loading pretrained BERT model
+    bertModel = TFBertModel.from_pretrained(MODEL)
```

Comment:
Use lower case camel case for variable names

Reply:
Used lower_case for all variable names.